modules/ibi-validated-software-versions.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+// * scalability_and_performance/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-prereqs_{context}"]
+= Software prerequisites for an image-based installation and deployment
+
+An image-based installation and deployment requires the following minimum software versions for these required components.
+
+.Minimum software requirements
+[cols=2*, width="80%", options="header"]
+|====
+|Component
+|Software version
+
+|Managed cluster version
+|4.14.13 or later
+
+|Hub cluster version
+|4.16
+
+|{rh-rhacm-first}
+|2.11
```

Comment:
Thanks for the comments @eranco74 - and the RHACM version should be 2.12? 

IBI Operator only packaged with MCE by default in 2.12